h/templates/includes/annotation_card.html.jinja2

```diff
@@ -29,5 +29,10 @@
   <section class="annotation-card__text">
     {{ annotation.text_rendered|safe }}
   </section>
+  <footer class="annotation-card__footer">
+    <a class="annotation-card__incontext-link" href="{{ incontext_link }}" target="_blank" title="{{ incontext_link }}">
```

Comment:
Did you mean to put the `"annotation-card__incontext-link"` CSS class on both the `<a>` and the `<svg>`?